locales/hi_IN/LC_MESSAGES/duckduckgo.po

```diff
@@ -173,6 +174,7 @@ msgid "Ad is suspicious"
 msgstr "विज्ञापन सन्देहजनक है"
 
 #. From Home page, options to "Add DuckDuckGo" to browser or set as home page.
+#, fuzzy,translated-trademark
 msgid "Add DuckDuckGo"
 msgstr "DuckDuckGo को लगाए"
```

Comment:
The translations in this file were fixed in a different PR. Looks like we can just drop the `#, fuzzy,translated-trademark` now?